groups/staffware-app/variables.tf

```diff
@@ -114,4 +114,10 @@ variable "cw_logs" {
   type        = map(any)
   description = "Map of log file information; used to create log groups, IAM permissions and passed to the application to configure remote logging"
   default     = {}
-}
\ No newline at end of file
+}
+
+variable "cloudwatch_namespace" {
+  type        = string
+  default     = null
+  description = "A custom namespace to define for CloudWatch custom metrics such as memory and disk"
+}
```

Comment:
As you're defaulting this value to `null`, you'll need to ensure the `cloudwatch_namespace` var is populated in the vars file for each profile.

Alternatively, you could set the default value of the variable if all profiles will use the same metrics namespace.

Reply:
After discussing with Jake, I've set a default value based on the value currently in use in both defined staffware-db roles - we can then see what it all looks like in staging and review as needed.